src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -5,10 +5,49 @@
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    public static final int MAX_AGE = 100;
+    public static final int MIN_AGE = 18;
+    public static final int MIN_LOGIN_LENGTH = 6;
+    public static final int MIN_PASSWORD_LENGTH = 6;
     private final StorageDao storageDao = new StorageDaoImpl();
 
     @Override
-    public User register(User user) {
-        return null;
+    public User register(User user) throws RegistrationUserException {
+        if (user == null) {
+            throw new RegistrationUserException("Please enter your data for registration!");
+        }
+        checkLogin(user);
+        checkPassword(user);
+        checkAge(user);
+        return storageDao.add(user);
+    }
+
+    private void checkLogin(User user) throws RegistrationUserException {
+        if (user.getLogin() == null) {
+            throw new RegistrationUserException("Please enter your login!");
+        } else if (user.getLogin().length() < MIN_LOGIN_LENGTH) {
```

Comment:
```suggestion
        if (user.getLogin().length() < MIN_LOGIN_LENGTH) {
```
you should not use `else` in case of  in`if` clause contains  exiting from the method.
fix all similar cases below in the code